actions/send-sves.vbs

```diff
@@ -199,22 +198,25 @@ ElseIf UNEA_radiobutton = 1 then
         claim_number = replace(claim_number, "_", "")
     End if
 
-    call navigate_to_MAXIS_screen("infc", "sves")
-    'checking for IRS non-disclosure agreement.
+    CALL navigate_to_MAXIS_screen("INFC" , "SVES")
```

Comment:
same question as above.

Reply:
it does not seem to have to go further into script